about.md

```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
 described in our [governance document](https://docs.google.com/document/d/1xjNzJqZjeRzUg5BMipQVUsy2xA1SRlEiyJLpPmUAs9E/edit?usp=sharing).
 
 
+## ESDS Committees and Working Groups
+
+### Leadership Committee
+* Dan Adriaansen, RAL
+* Drew Camron, Unidata
+* John Clyne, CISL
+* Katie Dagon, CGD (stepping down, ~Feb 2024)
+* Orhan Eroglu, CISL
+* Teagan King, CGD
+* Falko Judt, MMM
+* Ryan May, Unidata
+* Mike Levy, CGD
+* Ulrike Romatschk, EOL
```

Comment:
Maybe a bit nit-picky, but the first half of it feels like alphabetically ordered (per last names). If that's the case, the order of the last names in the second half needs to be Judt, King, Levy, May, Romatschk instead.

Reply:
Whoops. That's what happens when I multitask. Good catch. Thanks.